assets/less/content/general.less

```diff
@@ -101,18 +101,99 @@ h1 .note {
 
 blockquote {
   font-style: italic;
-  margin: .5em 0;
-  padding: .25em 1.5em;
   border-left: 3px solid @gray;
-  display: inline-block;
+  position: relative;
+  margin: 1.5625em 0;
+  padding: 0 1.2rem;
+  font-size: 1.28rem;
```

Comment:
```suggestion
```

I am thinking we should let the default font-size do its job so we keep everything consistent. WDYT? :)

Reply:
> I am thinking we should let the default font-size do its job so we keep everything consistent. WDYT? :)

Maybe I am wrong but does `rem` not just reflect a bigger font _relative_ to the default font size, or did you mean that this style should _have_ the default font size?

Reply:
I didn’t notice the rem but yeah, what I mean is to keep it as is. :)